packages/pdfkit/src/security.js

```diff
@@ -5,33 +5,557 @@
    By Yang Liu <hi@zesik.com>
  */
 
-// This file is ran directly with Node - needs to have .js extension
-// eslint-disable-next-line import/extensions
-import MD5 from 'crypto-js/md5.js';
-
-const wordArrayToBuffer = (wordArray) => {
-  const byteArray = [];
-
-  for (let i = 0; i < wordArray.sigBytes; i++) {
-    byteArray.push(
-      (wordArray.words[Math.floor(i / 4)] >> (8 * (3 - (i % 4)))) & 0xff
-    );
-  }
-
-  return Buffer.from(byteArray);
-};
+import CryptoJS from 'crypto-js';
```

Comment:
I wonder how this change affects bundle size and what was the original reason for selective import done like that.

Reply:
Yeah, I tried digging in revision history, but it was introduced with the selective import in https://github.com/diegomura/react-pdf/pull/1894. Upstream never used the selective import and imported the whole library from the beginning: https://github.com/foliojs/pdfkit/pull/820. I'm guessing it's because the full/upstream `security.js` file uses extensively this library - `CryptoJS.MD5`, `CryptoJS.RC4`, `CryptoJS.lib.WordArray`, `CryptoJS.SHA256`, `CryptoJS.AES`. Whereas the simplified version (not supporting password protection) indeed just used `CryptoJS.MD5`, so the selective import might have made sense for that use case.

For completeness sake (so that the implications of this PR on the bundle size are known):

#### Current `master`
```
646,480 pdfkit.browser.cjs
644,160 pdfkit.browser.js
326,577 pdfkit.browser.min.cjs
325,782 pdfkit.browser.min.js
407,839 pdfkit.cjs
405,488 pdfkit.js
232,858 pdfkit.min.cjs
231,996 pdfkit.min.js
```

#### This branch/PR
```
683,846 pdfkit.browser.cjs
680,853 pdfkit.browser.js
343,476 pdfkit.browser.min.cjs
342,353 pdfkit.browser.min.js
445,176 pdfkit.cjs
442,171 pdfkit.js
249,962 pdfkit.min.cjs
248,604 pdfkit.min.js
```

So, 9% increase for `pdfkit.js` and 7.2% for `pdfkit.min.js`.